diagnostics/app/controllers/DiagnosticsController.scala

```diff
@@ -40,9 +40,17 @@ object DiagnosticsController extends Controller with Logging {
     TinyResponse.noContent()
   }
 
-  def cssOptions = postOptions
+  def csp = Action(jsonParser) { implicit request =>
+    val shouldReport = (new Instant().getMillis) % 10000 == 1
```

Comment:
Correct me if I am wrong but this is only random if we assume this instruction timestamps are evenly distributed throughout time, which in reality is probably not the case. 
Is there a reason why this is not using `scala.util.Random`?